lib/remote.js

```diff
@@ -1,34 +1,43 @@
 'use strict';
 
-const unzip = require('node-unzip-2');
+const path = require('path');
+const fs = require('fs-extra');
+const unzip = require('adm-zip');
 const config = require('./config');
 const axios = require('axios');
 
 // Downloads the zip file from github and extracts it to folder
-exports.download = (path) => {
-  const url = config.get().repository;
+exports.download = (loc, lang) => {
+  // If the lang is english then keep the url simple, otherwise add language.
+  const url = config.get().repositoryBase + (lang === 'en' ? '' : '.' + lang) + '.zip';
+  const folderName = path.join(loc, 'pages' + (lang === 'en' ? '' : '.' + lang));
```

Comment:
Better to define our path suffix once and reuse it, make it easier to modify in the future:

```suggestion
  const suffix = lang === 'en' ? '' : '.' + lang;
  const url = config.get().repositoryBase + suffix + '.zip';
  const folderName = path.join(loc, 'pages' + suffix);
```